packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2577,6 +2577,12 @@ export default class Meeting extends StatelessWebexPlugin {
       );
     });
 
+    // We need to reinitialize  when user upgrades to host or cohost
+    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_MODERATOR_OR_COHOST_UPGRADE, (payload) => {
+      this.breakouts.queryPreAssignments(payload);
+      // ...
```

Comment:
question: Does the ... mean there is more to come here? What else will happen in this event handler?

Reply:
At present, this method handles just follow this process. when the attendee is upgraded to cohost or moderator, he can obtain the pre-assign list and operate the breakout session with high authority.